9/Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM ubuntu:16.04
 
-ADD config/redis.config /etc/redis/redis.config
-ADD start /start
+COPY config/redis.config /etc/redis/redis.config
```

Comment:
I moved the COPYs to the top, it does look a lot nicer and the files shouldn't change often. It's not much of an issue to develop with if you're good about commenting out blocks that you aren't working on.